src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const secondAccouning = (firstArg) => {
+    count++;
+
+    const sum = (secondArg) => {
+      if (count > 3 && count % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return firstArg + secondArg;
+      }
```

Comment:
There is no need in `else` statement. And i think using ? and : will be better in this case.